### PR TITLE
Chore: Remove all ifdef checks

### PR DESCRIPTION
**Pull request by KhaledChehabeddine**
- Removing ifdef checks in EX2Kit

### CODE REVIEW

File: EX2Kit/EX2Kit.h [hunk 1]
```diff
@@ -18,10 +18,8 @@
     #import "DDLog.h"
     #import "UIDevice+Hardware.h"
 #else
-    #ifdef IOS
```
Comment (AmerAbb) [inline EX2Kit/EX2Kit.h:21]: dont think we can remove them here

File: EX2Kit/Components/EX2Components.h [hunk 2]
```diff
@@ -18,9 +18,7 @@
 #import "EX2Action.h"
 #import "EX2ActionBlock.h"
 
-#ifdef IOS
```
Comment (AmerAbb) [inline EX2Kit/Components/EX2Components.h:21]: wala hon

File: EX2Kit/Categories/EX2Categories.h [hunk 3]
```diff
@@ -9,10 +9,8 @@
 #ifndef EX2Kit_Categories_h
 #define EX2Kit_Categories_h
 
-#ifdef IOS
```
Comment (AmerAbb) [inline EX2Kit/Categories/EX2Categories.h:12]: wala hon

**Comment by AmerAbb** [comment]
the tv and watchos are targets in ex2kit

**Comment by KhaledChehabeddine** [comment]
So EX2Kit in general should be left untouched?

**Pull request closed without merging**